components/gsl/Makefile

```diff
@@ -9,36 +9,39 @@
 #
 
 #
-# Copyright (c) 2013 Louis M. Picciano. All rights reserved.
+# Copyright (c) 2013-2016 Louis M. Picciano. All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include $(WS_TOP)/make-rules/shared-macros.mk
```

Comment:
Actually, it is quite used (and @alp - you could see a lot of it in my recent commits). This facilitates especially the restructuring of directory layout you do now - the amount of parent ".." components has to only be fixes for the shared-macros; the rest of included makefiles are in a fixed location under now-known WS_TOP. So I'd rather convert other Makefiles to use it, as part of the effort to change directory locations or otherwise a best practice ;)